packages/core/app-layout/src/components/pageHeader/AppPageHeader.cy.ts

```diff
@@ -0,0 +1,84 @@
+// Cypress component test spec file
+
+import AppPageHeader from './AppPageHeader.vue'
+
+describe('<AppPageHeader />', () => {
```

Comment:
Since the breadcrumb icon slot logic is directly tied to the KBreadcrumbs component, there should be test coverage here to ensure the slot content renders correctly

Reply:
I already added it... https://github.com/Kong/public-ui-components/pull/610/files#diff-37753584988831e9fb98576d5feac57c4849ede0716863f9e9efe4b62e311c98R56